src/js/views/Project/GitlabImport.jsx

```diff
@@ -18,7 +18,7 @@ function GitlabImport() {
   }
   const [errors, setErrors] = useState(emptyErrors)
   const [state] = useContext(Context)
-  const history = useHistory()
+  const history = useNavigate()
```

Comment:
You need to change the usage from `history.push(...)` to `history(...)` since `useNavigate` returns a function.  I recommend renaming `history` to `navigate`.

```javascript
  const navigate = useNavigate()
  // ...
  navigate(`/ui/projects/${projectId}`)
```
instead of 
```javascript
  const history = useHistory()
  // ...
  history.push(`/ui/projects/${projectId}`)
```

https://github.com/AWeber-Imbi/imbi-ui/blob/010960c3736352016455b7e13bdd3cf321b9e45b/src/js/views/Project/GitlabImport.jsx#L135

Reply:
The same error exists in:
* [src/js/index.jsx](https://github.com/AWeber-Imbi/imbi-ui/blob/010960c3736352016455b7e13bdd3cf321b9e45b/src/js/index.jsx#L68)
* [src/js/views/Dashboard/Namespaces/Namespaces.jsx](https://github.com/AWeber-Imbi/imbi-ui/blob/010960c3736352016455b7e13bdd3cf321b9e45b/src/js/views/Dashboard/Namespaces/Namespaces.jsx#L204-L206)
* [src/js/views/Project/Create/Create.jsx](https://github.com/AWeber-Imbi/imbi-ui/blob/010960c3736352016455b7e13bdd3cf321b9e45b/src/js/views/Project/Create/Create.jsx#L154)
* _src/js/views/Project/Settings.jsx_ in [onArchive](https://github.com/AWeber-Imbi/imbi-ui/blob/010960c3736352016455b7e13bdd3cf321b9e45b/src/js/views/Project/Settings.jsx#L50) as well as [onDelete](https://github.com/AWeber-Imbi/imbi-ui/blob/010960c3736352016455b7e13bdd3cf321b9e45b/src/js/views/Project/Settings.jsx#L66)
* [src/js/vies/Projects/Projects.jsx](https://github.com/AWeber-Imbi/imbi-ui/blob/010960c3736352016455b7e13bdd3cf321b9e45b/src/js/views/Projects/Projects.jsx#L84-L86)